### PR TITLE
Yea 123. Unsubscribe flow

### PR DESCRIPTION
**Pull request by Ledespana**
Screenshot 2024-01-22 at 4 51 36 PM

### CODE REVIEW

File: frontend/components/JoinGraveBtn.tsx [hunk 1]
```diff
@@ -50,6 +52,7 @@ export default function JoinGraveBtn({
     setBrowserExtensionControllerAddress,
   ] = useState<string>('');
   const [joiningStep, setJoiningStep] = useState<number>(0);
+  const [leavingStep, setLeavingStep] = useState<number>(-1);
```
Comment (democrazyfi) [inline frontend/components/JoinGraveBtn.tsx:55]: why not 0 too?

File: frontend/components/JoinGravePanel.tsx [hunk 2]
```diff
@@ -101,11 +115,119 @@ const JoinGravePanel: React.FC = () => {
     setActiveStep(newStep);
   };
 
+  const handleLeavingStep = (leavingStep: number, data: any) => {
```
Comment (democrazyfi) [inline frontend/components/JoinGravePanel.tsx:118]: data param unused here

**Comment by tehnalogos** [comment]
@Ledespana some merge conflicts

File: frontend/components/JoinGraveBtn.tsx (hunk 1 above)
Comment (Ledespana) [reply frontend/components/JoinGraveBtn.tsx:55]: Because it is hidden. Step number 0 makes the flow to be displayed

**Review by tehnalogos — APPROVED** [review]

**Pull request merged**